cmd/serve_migrate.go

```diff
@@ -0,0 +1,71 @@
+package cmd
+
+import (
+	"database/sql"
+	"fmt"
+	"io/fs"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+
+	"github.com/charmbracelet/charm/server"
+	"github.com/charmbracelet/charm/server/db/sqlite"
+	"github.com/spf13/cobra"
+	_ "modernc.org/sqlite"
+)
+
+var (
+	ServeMigrationCmd = &cobra.Command{
+		Use:     "migrate",
+		Aliases: []string{"migration"},
+		Hidden:  false,
+		Short:   "Run the server migration tool.",
+		Long:    paragraph("Run the server migration tool to migrate the database."),
+		Args:    cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cfg := server.DefaultConfig()
+			dp := filepath.Join(cfg.DataDir, "db", sqlite.DbName)
+			_, err := os.Stat(dp)
+			if err != nil {
+				return fmt.Errorf("database does not exist: %s", err)
+			}
+			log.Printf("Opening SQLite db: %s\n", dp)
+			db, err := sql.Open("sqlite", dp+sqlite.DbOptions)
+			if err != nil {
+				return err
+			}
+			err = filepath.Walk("migrations", func(path string, info fs.FileInfo, err error) error {
```

Comment:
we probably need to bundle the migration sql binary since they won't have these files when they download `charm`.